src/screens/User/User.tsx

```diff
@@ -1,33 +1,40 @@
-import React from 'react'
+import React, { useEffect } from 'react'
 import { View, Image, TouchableOpacity, Linking, FlatList } from 'react-native'
-import { Heading, HeadingType, PlainText } from 'src/components'
-import { Podcast, User } from 'src/common/types'
+import { useDispatch } from 'react-redux'
 import PodcastItem from './components'
-import { MockedUser, MockedPodcasts } from './mockedData'
+import { UserScreenNavigationProp } from './common'
+import { useAppSelector } from 'src/hooks'
+import { RootState } from 'src/common/types'
+import { resetUser, loadPodcasts } from 'src/store/actions'
+import { Heading, HeadingType, PlainText } from 'src/components'
+import { Podcast } from 'src/common/types'
 import Check from 'src/assets/images/checkMark.svg'
 import AtMark from 'src/assets/images/atMark.svg'
 import LogOut from 'src/assets/images/iconmonstr-log-out-16.svg'
 import styles from './styles'
 
 type Props = {
-  podcasts: Podcast[]
-  user: User
+  navigation: UserScreenNavigationProp
 }
 
-const userPageInfo: Props = {
-  podcasts: MockedPodcasts,
-  user: MockedUser
-}
+const User: React.FC<Props> = ({ navigation }) => {
+  const dispatch = useDispatch()
 
-const UserPage: React.FC<Props> = () => {
   const handleLogOut = () => {
-    return 0
+    dispatch(resetUser())
   }
 
+  const { user, podcasts } = useAppSelector(({ auth, user }: RootState) => ({
+    user: auth.user,
+    podcasts: user.podcasts
+  }))
+
+  useEffect(() => {
+    dispatch(loadPodcasts(Number(user?.id)))
+  }, [user?.id])
+
   const handleOpenMail = async () => {
-    await Linking.openURL(
-      `mailto://${userPageInfo.user.email}}&subject=abcdefg&body=body`
-    )
+    await Linking.openURL(`mailto://${user?.email}}`)
   }
 
```

Comment:
Let's add here a preloader, check if user exists and avoid `.?`
```suggestion
if (isLoading) {
    return (
      <View>
        <ActivityIndicator size="large" color="#f3427f" />
      </View>
    )
  }
  
  if(!user) {
      return null;
  }
```